src/main/java/de/zalando/aruha/nakadi/domain/EventType.java

```diff
@@ -2,30 +2,60 @@
 
 import java.util.List;
 
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonSetter;
 import com.google.common.collect.Lists;
 
 public class EventType {
```

Comment:
You're right @v-stepanov :+1: 

Just simplified things a little here https://github.com/zalando/nakadi/commit/8d2df1cfff15cc78f4a7963fa3c999012ff239cd